english/explore-outside/journey-to-flaming-tower/walk/portal/dinner/dinner.md

```diff
@@ -1,3 +1,10 @@
 As you walk with this imp you feel yourself
-grow ever shorter untill you are his size. You
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.